### ISSUE TITLE
qtwebkit build is aborted because of log size

### ISSUE THREAD

**Issue by FRidh** [NixOS/nixpkgs#68341]
**Describe the bug**
qtwebkit fails to build. This impacts the whole of KDE.

**Metadata**
https://hydra.nixos.org/build/100190178

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: qt5.qtwebkit
# a list of nixos modules affected by the problem
module:
```

**Comment by 06kellyjac** [comment]
After some investigation @ the NixOS meetup we've noticed that Hydra seems the be building the wrong package as the logs are referring to building `qtwebkit` but it should actually be building `messagelib`.

One idea is that the mirror for `messagelib` has the wrong source. The other is that Hydra is messed up.

The changes between `.0` & `.1` for `messagelib` (between it working and it breaking) are very minor and don't seem to reference `qtwebkit` in the changes.

I've tried taking the release build for `messagelib` (re-archiving `.tar.gz` -> `.tar.xz`) and comparing the sha256 to the one in the KDE folder's `srcs.nix` and they do not match however my sha doesn't match anything in `srcs.nix` and it's easy for my release from the KDE GH to be different to the mirror anyway.
My SHA: `f6bbe0726ac821bb9bd352e1ac0e68f14d78b0e5dab2f03cff55adc92613fd5f`
Nixpkgs SHA: `ec43d913028124a49eaa440e4b55dba23b6ab503728b897a3ad8e1fc5e446802`
